src/components/CodeAssignmentModal/index.jsx

```diff
@@ -18,8 +19,9 @@ import { ONCE_PER_CUSTOMER, MULTI_USE, CSV_HEADER_NAME } from '../../data/consta
 import { EMAIL_ADDRESS_TEXT_FORM_DATA, EMAIL_ADDRESS_CSV_FORM_DATA } from '../../data/constants/addUsers';
 
 import './CodeAssignmentModal.scss';
+import { configuration } from '../../config';
 
-class CodeAssignmentModal extends React.Component {
+class BaseCodeAssignmentModal extends React.Component {
```

Comment:
sorry if I missed prior context, the reason for renaming this to Base* ?

Reply:
Because I'm connecting it to Redux, and our linter rules won't let me rename the connected component the same thing as the base component, and I prefer this pattern to calling the connected component `ConnectedCodeAssignmentModel`. I'll often use the Base class for testing, because it takes away the complexity of the redux provider, though I didn't do that in this case.

Reply:
Ah I see, but I guess I did not see what was overriding or subclassing it, so did not see the 'Base'-ness of it is what I had meant ..